dev/langtool/meta/ckeditor.core/meta.txt

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
-# For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+# For licensing, see LICENSE.md or http://ckeditor.com/license
```

Comment:
Looks like some issue with `langtool` and some outdated headers, cc @Comandeer

Reply:
It seems to be an issue with the langtool itself, as I don't see such header anywhere in the code 🤔

Reply:
Yeah, something is wrong. I checked what `langtool` did exactly and it turns out that besides messing up license header it also added trailing spaces after `=` sign and changed the order of entries from alphabetical to some random one (probably there are some kind of rules but I don't see them). So instead of using this tool I just added the necessary entry manually.